backend/molgenis-emx2/src/main/java/org/molgenis/emx2/TableMetadata.java

```diff
@@ -157,9 +157,7 @@ public void setSchema(SchemaMetadata schemaMetadata) {
   }
 
   public List<Column> getColumnsWithoutConstant() {
```

Comment:
rename to getColumnsWithoutHeadings for be consistent with the rest of the changes